src/main/java/com/example/ufo_fi/domain/tradepost/dto/request/TradePostUpdateReq.java

```diff
@@ -0,0 +1,21 @@
+package com.example.ufo_fi.domain.tradepost.dto.request;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+
+@Getter
+@NoArgsConstructor
+public class TradePostUpdateReq {
+
+    @Size(min = 1, max = 15, message = "제목은 1~15자 이내여야 합니다.")
+    private String title;
```

Comment:
```suggestion
    @NotEmpty(message = "제목은 필수값입니다.")
    @Size(min = 1, max = 15, message = "제목은 1~15자 이내여야 합니다.")
    private String title;
```
유효하지 않은 값이 들어올 가능성에 대해 고민해보시면 좋을 것 같습니다.